source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/test/eventbridge-kinesisfirehose-s3.test.js

```diff
@@ -0,0 +1,142 @@
+"use strict";
```

Comment:
Build product, shouldn't be in git. Seems like you're miss *.js in your .gitignore file. The original .gitignore starts with:

```
lib/*.js
test/*.js
!test/lambda/*
*.js.map
```

This construct is missing the first 3 lines. The 3rd line is not needed as there is no test/lambda folder in this construct (I think), but the first 2 lines are very neccessary.